src/core/reference/include/openvino/reference/identity.hpp

```diff
@@ -18,9 +20,23 @@ namespace reference {
 static inline void identity(const char* input, char* output, const size_t size_in_bytes) {
     if (input == output) {
         return;
-    } else {
-        std::memcpy(output, input, size_in_bytes);
     }
+    std::memcpy(output, input, size_in_bytes);
 }
+
+/**
+ * @brief Identity operation computes the identity of the input tensor.
+ *
+ * @param input Input matrix (matrices) pointer.
+ * @param output Output matrix (matrices) pointer.
+ * @param shape_size Size of the input tensor shape.
+ **/
+static inline void identity(const std::string* input, std::string* output, const size_t shape_size) {
+    if (input == output) {
+        return;
+    }
+    std::copy_n(input, shape_size, output);
```

Comment:
Please update reference test cases to test strings: src/plugins/template/tests/functional/op_reference/identity.cpp